packages/fields-document/src/structure-graphql-input.tsx

```diff
@@ -442,7 +442,7 @@ export async function checkUniqueItemExists (
     throw missingItem(operation, uniqueInput)
   }
 
-  return { id: item.id.toString() }
+  return { id: item.id }
```

Comment:
Removing this `toString` will break if you have a bigint since a bigint wouldn't be accepted by the `ID` type